### DISCUSSION TITLE
Projetos similares

### DISCUSSION THREAD

**Discussion by george-gca** [levxyca/diciotech#180] [General]
Achei interessante adicionar aqui alguns projetos com uma proposta igual ou parecida com esse. De repente até podemos pegar algumas ideias de alguns deles e implementar aqui.

Por exemplo, nesse aqui eles adicionaram um link no final das explicações com uma referência de onde ler mais sobre. Acho interessante.

https://github.com/Buzzpy/Dev-Encyclopedia

**Comment by levxyca** [comment]
@george-gca boooa! com certeza é interessante sim, não conhecia esse projeto, já vou dar um bizu pra ver se tiramos uns insights legais!

**Comment by george-gca** [comment]
Outro projeto similar e interessante é o https://the-algorithms.com/. O foco desse é em algoritmos e suas implementações em várias linguagens de programação.
